code.json

```
@@ -50,10 +50,10 @@
         }
       ]
     },
-    "homepageURL": "",
-    "downloadURL": "",
-    "disclaimerURL": "",
-    "repositoryURL": "",
+    "homepageURL": "https://github.com/USGS-R/ds-pipelines-targets-example-wqp/releases/tag/v0.1.0",
+    "downloadURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/archive/refs/tags/v0.1.0.zip",
+    "disclaimerURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/blob/main/DISCLAIMER.md",
```

@padilla410, would it make sense to point these url's to USGS-R for now (as you do with the `homepageURL` ) and then update all lines when we make the transition? (Feel free to just ignore this comment since I don't have all the context for DGEC stuff 😸)

---

I am curious about this, too. When I create the migration-assistant PR, I don't have the ability to correctly populate these fields with future URLs, so I've been doing the current URL at the time I create the PR. Is that OK? In either case, I think it would be best to be consistent, and either go with all USGS-R or all DOI-USGS.

---

This is a typo!!! Nice catch yall

---

Also, I want to point these to USGS-DOI because I am trying to get this in right now

---

@padilla410 and I decided that we will go ahead and use the USGS-DOI links for now even though they don't actually work yet. We'll merge some planned code changes and then go through with the migration in ~1-2 weeks.

---

I'm under the impression that the url's need to point to the raw files (e.g. no 'blob' in the file path). But the `snow-to-flow` repo in DGEC uses the rendered url's for both the license and the disclaimer, so I'll let you decide whether you want to see if it goes through approval as is. 

```suggestion
    "disclaimerURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/raw/main/DISCLAIMER.md",
```